AmazonChimeSDK/AmazonChimeSDK/device/DefaultDeviceController.swift

```diff
@@ -37,8 +37,20 @@ import AVFoundation
     public func listAudioDevices() -> [MediaDevice] {
         var inputDevices: [MediaDevice] = []
         let loudSpeaker = getSpeakerMediaDevice()
-        if let availablePort = audioSession.availableInputs {
-            inputDevices = availablePort.map { port in MediaDevice.fromAVSessionPort(port: port) }
+        if let availableInputs = audioSession.availableInputs {
+            // On iOS 15, availableInputs returns two entries for bluetooth devices.
+            // All fields are identical other than uid and portType.
```

Comment:
There are duplicate entries for 1 Bluetooth device. Does it matter which entry we choose to pick and add into the `inputDevices`?

Reply:
No, audio works no matter which one you choose